### PR TITLE
Style fixes and documentation

### PR DESCRIPTION
**Pull request by mprahl**
Review commit by commit for a rundown of the changes made

### CODE REVIEW

File: purview/api/v1.py [hunk 1]
```diff
@@ -17,14 +17,25 @@
 @api_v1.route('/about')
 def about():
     """
-    Display version info about purview
-    :return: A JSON object with version info
+    Display general information about the app.
```
Comment (yashvardhannanavati) [inline purview/api/v1.py]: How about "...about the API" ?

File: purview/app.py [hunk 2]
```diff
@@ -15,6 +15,11 @@
 
 
 def load_config(app):
+    """
+    Determine the correct configuration to use and apply it.
```
Comment (yashvardhannanavati) [inline purview/app.py]: You can remove the "and apply it" part if you want. It seems like extra

File: purview/config.py [hunk 3]
```diff
@@ -4,9 +4,8 @@
 
 
 class Config(object):
-    """
-    A Purview Flask configuration
-    """
+    """The base Purview application configuration."""
```
Comment (yashvardhannanavati) [inline purview/config.py]: Any specific reason to have the quotes and the comment in a single line? If not then maybe we can have a uniform pattern throughout

File: README.md [hunk 4]
```diff
@@ -1,3 +1,72 @@
 # Purview API
-An API for Purview
+The API and scrapers for Purview
```
[the remark this answers is not in the document]
Comment (mprahl) [reply README.md]: Sure

**Comment by mprahl** [comment]
@yashvardhannanavati fixed. Is this good to merge now?

File: purview/models/distgit.py [hunk 5]
```diff
@@ -38,6 +44,8 @@ class DistGitPush(PurviewStructuredNode):
 
 
 class DistGitCommit(PurviewStructuredNode):
+    """A representation of a dist-git commit in Neo4j."""
```
Comment (yashvardhannanavati) [inline purview/models/distgit.py]: "Definition of a dist-git branch in Neo4j" ?

Comment (yashvardhannanavati) [reply purview/models/distgit.py]: same for all other classes

File: purview/utils/general.py [hunk 6]
```diff
@@ -10,9 +10,12 @@
 
 def timestamp_to_datetime(timestamp):
     """
-    Parses a timestamp and return a datetime object
-    :param timestamp: a string of a timestamp
-    :return: a datetime object
+    Convert a string timestamp to a datetime object.
+
+    :param str timestamp: a timestamp
```
Comment (yashvardhannanavati) [inline purview/utils/general.py]: "string in timestamp format"?

File: purview/api/v1.py (hunk 1 above)
Comment (mprahl) [reply purview/api/v1.py]: Well we're actually exposing the app's version not the API so I think it's good as is.

File: purview/app.py (hunk 2 above)
Comment (mprahl) [reply purview/app.py]: I don't think it's implied by the function name so I'll keep it.

File: purview/config.py (hunk 3 above)
Comment (mprahl) [reply purview/config.py]: If you have a docstring that is just one line of text, it has to be one line like that based on the PEP guidelines.

File: purview/utils/general.py (hunk 6 above)
Comment (mprahl) [reply purview/utils/general.py]: Changing to `:param str timestamp: a generic or ISO-8601 timestamp`

**Comment by yashvardhannanavati** [comment]
:1st_place_medal:

**Pull request merged**